### PR TITLE
Update RNViewOverflowManager ( borderRadius issue )

### PR DESCRIPTION
**Pull request by ysfzrn**
Setting borderRadius with style object

### CODE REVIEW

File: android/src/main/java/com/entria/views/RNViewOverflowManager.java [hunk 1]
```diff
@@ -16,5 +17,10 @@ protected RNViewOverflowLayout createViewInstance(ThemedReactContext reactContex
     public String getName() {
         return REACT_CLASS;
     }
+    
+    @ReactProp(name = "borderRadius", defaultFloat = 0f)
+    public void setBorderRadius(RNViewOverflowLayout view, float borderRadius) {
+      view.setBorderRadius(borderRadius*3.5f);
```
Comment (sibelius) [inline android/src/main/java/com/entria/views/RNViewOverflowManager.java]: 3.5f should be in user land?

Comment (ysfzrn) [reply android/src/main/java/com/entria/views/RNViewOverflowManager.java]: Sorry, it is unnecessary. I'm going to send another PR

Comment (ysfzrn) [reply android/src/main/java/com/entria/views/RNViewOverflowManager.java]: I removed multiplier

**Comment by sibelius** [comment]
tks

**Pull request merged**